test/pages/Inventory.page.js

```diff
@@ -9,6 +9,39 @@ class InventoryPage extends BaseSwagLabPage {
 
     get addItemToCartBtns() { return $$('[id^="add-to-cart"]'); }
 
+    get inventoryItemsNames() { return $$('.inventory_item_name'); }
+
+    get inventoryItemsPrices() { return $$('.inventory_item_price'); }
+
+    get sort() { return $('.product_sort_container'); }
+
+    get activeOption() { return $('.active_option'); }
+
+    // getSortItem(value) {
+    //     return $(`.product_sort_container option[value="${value}"]`);
+    // }
+    getSortItem(value) {
+        return $('.product_sort_container').$(`option[value="${value}"]`);
```

Comment:
.product_sort_container = this.sort?

Reply:
So this is a drop-down list selector in which I then look for an option by which I choose sorting

Reply:
I mean you can replace '.product_sort_container' in this string with this.sort